CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+Each PR has to be approved by at least one reviewer before merging it. A person can be assigned as reviewer by either mark them as such or asking for a review by tagging the person in the description/comment of the PR.
+
+A draft pull request can be made even if the branch is not ready to be merged yet. This way people/reviewer know that there is someone still working on that branch actively. This gives them the opportunity share their thoughts knowing that the pull request is still subject to change and does not need a full review yet.
+
+The [CHANGELOG.md](CHANGELOG.md) describes changes made in a pull request. It should contain a short description of the performed changes, as well as (a) link(s) to issue(s) or pull request.
+
+#### Review Process
+
+1. Dev makes a PR.
+2. Rev reviews and raises some comments.
+3. Dev addresses the comments and leaves responses explaining what has to be done. In cases where Dev just implemented Rev's suggestion, a simple "Done" is sufficient.
+4. Rev reviews the changes and
+    - If Rev is happy with a change, then Rev resolves the comment.
+    - If Rev is still unsatisfied with a change, then Rev adds another comment explaining what is still missing.
+5. Restart from 3 until all comments are resolved.
+
+### Git Workflow
+
+All development work is based on the main branch (`main`). Pull requests are expected to target the `main` branch.
+
+## Style Guide
+
+### Python
+
+In short:
+- Guideline: [PEP8](https://peps.python.org/pep-0008/), [PEP 484 (Type Hints)](https://peps.python.org/pep-0484/)
+- Linter: [flake8](https://flake8.pycqa.org), [Pylint](https://pylint.pycqa.org/)
+
+#### Linters
+
+This project uses [flake8](https://flake8.pycqa.org) and [Pylint](https://pylint.pycqa.org/) to ensure consistent code style.
+
+Run flake8 and Pylint with following commands:
+
+```bash
+flake8 .
+pylint .
+```
+
+*Tips:*
+- Mark in-line that flake8 should not raise any error: `print()  # noqa`
+
+### HTML
+
+[.htmlhintrc](.htmlhintrc) and the [documentation of these rules](https://github.com/htmlhint/HTMLHint/tree/master/docs/user-guide/rules)
+
+### JavaScript
+
+[Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript) with some modifications (see [.eslintrc.json](.eslintrc.json))
+
+### CSS
+
+[stylelint-config-standard](https://github.com/stylelint/stylelint-config-standard) with some modifications (see [.stylelintrc.json](.stylelintrc.json))  
+
+## Commit Hooks
+
+If you discover any violations in the existing code, feel very welcome to fix them. To facilitate paying attention to these conventions, please make sure that for your pull request all checks succeed. You can also set up git hooks to automatically run relevant linters before your code is committed, see [.hooks/README.MD](.hooks/README.MD) for more info. Please also make sure that everything you modify or add is covered by [unit tests](test).
+
+## Tests
+
+Please provide tests.
```

Comment:
I think we should specify which frameworks we use for the tests and also which kinds of tests, e.g. unit tests with pytest. System tests are not done automatically yet, but we could already think of how it should be done in the future

Reply:
Okay. What do you suggest here? Just ask for unit tests using the Pytest framework?